product_docs/docs/biganimal/release/using_cluster/05b_major_version_upgrade.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Upgrading Postgres major versions
+navTitle: Upgrading Postgres major versions
+---
+
+To upgrade small to moderately large databases (up to several tens of gigabytes) between major versions of Postgres on BigAnimal, we recommend utilizing pg_dump to migrate data and populate a new BigAnimal instance/cluster. This method will require some downtime, so schedule the pg_dump operation accordingly.
```

Comment:
Refer to https://www.enterprisedb.com/docs/biganimal/latest/migration/cold_migration/  topic to be aligned with font treatment for the utility names or any other keywords. We need to make the changes across the topic.

```suggestion
To upgrade small to moderately large databases (up to several tens of gigabytes) between major versions of Postgres on BigAnimal, we recommend utilizing `pg_dump` to migrate data and populate a new BigAnimal instance/cluster. This method requires downtime, so schedule the `pg_dump` operation accordingly.
```